src/app/zap-templates/zcl/custom-types.xml

```diff
@@ -42,4 +43,10 @@ limitations under the License.
      <item name="type" type="DEVICE_TYPE_ID"/>
      <item name="revision" type="INT16U"/>
    </struct>
+  <struct name="FabricDescriptor">
+     <item name="FabricId" type="FABRIC_ID"/>
+     <item name="VendorId" type="INT16U"/>
+     <item name="label" type="OCTET_STRING" length="32"/>
```

Comment:
Why OCTET_STRING and not CHAR_STRING?  I'd expect the latter; this is meant to be text data.

Reply:
Vivien told me to use OCTET_STRING because CHAR_STRING is not supported first (that is what I had done first)

Reply:
Ugh.  @vivien-apple is there an issue tracking CHAR_STRING support?  Should we have an issue here to make this a CHAR_STRING, with a comment pointing to it?

Reply:
Filed an issue and added a comment: https://github.com/project-chip/connectedhomeip/issues/6112

I initially wanted to use CHAR_STRING but it requires a bit more code to work properly.

Reply:
> Ugh. @vivien-apple is there an issue tracking CHAR_STRING support? Should we have an issue here to make this a CHAR_STRING, with a comment pointing to it?

FTR, `CHAR_STRING` works for most use cases. But I'm not 100% confident that `List[T]` supports it yet. I need to check.